src/cupbearer/detectors/mahalanobis/__init__.py

```diff
@@ -1,24 +1,26 @@
 from dataclasses import dataclass, field
 
-from cupbearer.detectors.config import ActivationBasedDetectorConfig, TrainConfig
+from cupbearer.detectors.config import ActivationBasedDetectorConfig
+from cupbearer.utils.utils import BaseConfig
 
 from .mahalanobis_detector import MahalanobisDetector
 
 
 @dataclass
-class MahalanobisTrainConfig(TrainConfig):
-    max_batches: int = 0
+class MahalanobisTrainConfig(BaseConfig):
+    max_batches = 0
```

Comment:
Was removing the type hint here deliberate?

Reply:
Oops, don't know how that could have happened. I probably should start reviewing my own code as well...